package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "iot-solutions",
-  "version": "1.0.10",
```

Comment:
Do we need to update package-lock.json? I don't see any other changes in package.json